cheats/rules/partial-object/cheat.rego

```diff
@@ -0,0 +1,10 @@
+package cheat
+
+import rego.v1
```

Comment:
Why? 🙂

Reply:
I'll freely admit that I was borrowing from the other examples on that. Will refactor shortly.